File: exs/mundo_1/ruby/019.rb
=begin
Desafio 019

Problema: Um professor quer sortear um dos seus quatro alunos para
          apagar o quadro. Faça um programa que ajude ele, lendo o
          nome dos alunos e escrevendo na tela o nome do escolhido.

Resolução do problema:
=end

alunos = []
4.times do
	print"Digite o nome do aluno: "
	a = gets.chomp
	alunos.push(a)
end

aluno_aleatorio = alunos.sample
puts"O aluno que irá apagar a lousa é o(a) #{aluno_aleatorio}."

File: exs/mundo_1/golang/019.go
/*
Desafio 019

Problema: Um professor quer sortear um dos seus quatro alunos para
          apagar o quadro. Faça um programa que ajude ele, lendo o
          nome dos alunos e escrevendo na tela o nome do escolhido.

Resolução do problema:
*/

package main


import(
	"fmt"
	"math/rand"
	"time"
)


var aluno string

func main(){
	var alunos  []string
	for i := 0; i < 4; i += 1{
		fmt.Print("Digite o nome do aluno: ")
		fmt.Scanln(&aluno)
		alunos = append(alunos, aluno)
	} 

	rand.Seed(time.Now().Unix())
	fmt.Println("O aluno escolhido foi o(a)", alunos[rand.Intn(len(alunos))])
}
